### PR TITLE
redux example charts listing

### PR DESCRIPTION
**Pull request by spasovski**
We'll really need to finalize the JSON structure but this should be a start to populating fake data for MG.

### CODE REVIEW

**Comment by openjck** [comment]
Looks like this needs a rebase :frowning:

File: distributionviewer/core/static/js/app/api/chart-api.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  GET_CHARTS: 'http://localhost:3009/charts'
+};
+
+const prodEndpoints = {
+  // TODO: Likely subject to change.
+  GET_CHARTS: 'https://moz-distribution-viewer.herokuapp.com/charts'
+}
+
+// TODO: Check node environment to set this.
+const endpoints = mockEndpoints;
```
Comment (openjck) [inline distributionviewer/core/static/js/app/api/chart-api.js:15]: Good catch. We'll need to do that for the site to work on Heroku.

File: db.json [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        "id": 3,
+        "name": "hudsons robbed"
+      },
+      {
+        "id": 4,
+        "name": "short allens"
+      },
+      {
+        "id": 5,
+        "name": "kara de halis"
```
Comment (openjck) [inline db.json:22]: :heart:

File: distributionviewer/core/static/js/app/components/views/example-redux-chart.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+export default function(props) {
+  return (
+    <pre>
+      <code>
+        {props.charts.map(chart => {
+          return (<p>{chart.name}</p>);
+        })}
+      </code>
+    </pre>
```
Comment (openjck) [inline distributionviewer/core/static/js/app/components/views/example-redux-chart.js]: I notice we're using &lt;pre&gt; and &lt;code&gt; here. Is that for something down the road?

Comment (spasovski) [inline distributionviewer/core/static/js/app/components/views/example-redux-chart.js]: I meant to spit out the raw JSON and react threw a hissy fit. I should've nuked the wrapping markup.

File: gulpfile.js [hunk 4]
```diff
@@ -36,5 +37,11 @@ gulp.task('watch', ['build'], function() {
   gulp.watch(path.resolve(CSS, '**/*.styl'), ['css']);
 });
 
+// Fake API
+gulp.task('serve:api', function(done) {
+  cp.exec('node_modules/.bin/json-server --watch db.json --port 3009', {stdio: 'inherit'})
```
Comment (openjck) [inline gulpfile.js]: Not a big deal, but we can optionally leave off the path (`node_modules/.bin/json-server` -> `json-server`) and Node will find the path on its own.

**Pull request merged**